internal/image_template.go

```diff
@@ -33,8 +34,25 @@ func (t *ImageTemplate) String() string {
 	return t.Raw
 }
 
+func prepTemplateString(input string) string {
```

Comment:
Some information about what this method does and why is necessary would be useful